src/nnue/layers/affine_transform_sparse_input.h

```diff
@@ -294,18 +304,38 @@ class AffineTransformSparseInput {
         // convince GCC to not do weird pointer arithmetic in the following loop
         const std::int8_t* weights_cp = weights;
 
+        if constexpr (SplitAccums)
+        {
+            acc[1] = vec_set_32(0);
+            while (start + 1 < end)
+            {
+                std::ptrdiff_t i1  = *start++;
```

Comment:
Maybe the style here and lines 328-329 should match just for consistency.
```const std::ptrdiff_t i1 = *start++;``` in both places?

Reply:
yes sure!

Reply:
yessir